docs/lakehouse-iceberg-catalog.md

```diff
@@ -111,10 +111,28 @@ For `bucket` and `truncate`, the first argument must be integer literal, and the
 
 ### Table distributions
 
-- Doesn't support `Distribution`, you should use `BucketPartition` instead.
+- Gravitino used by default `NoneDistribution`. Hash distribute by partition key.
+```json
+{
+  "strategy": "hash"
+}
+```
+```java
+Distributions.hash();
+```
+
+- Support `RangeDistribution`, You can pass `range` as values through the API. Range distribute by partition key or sort key if table has an SortOrder.
+```json
+{
+  "strategy": "range"
+}
+```
+```java
+Distributions.range();
+```
 
 :::info
-If you load Iceberg tables, the table distribution strategy is `hash` with num 0, which means no distribution.
+Iceberg automatically calculates the bucket value and performs hashing based on the partition field, or automatically calculates ranges according to SortOrder.
```

Comment:
suggest to add something like no support add expressions to distribution for Iceberg.